assets/node.yaml

```diff
@@ -44,8 +44,12 @@ spec:
               mountPropagation: "Bidirectional"
             - name: plugin-dir
               mountPath: /csi
-            - name: device-dir
-              mountPath: /dev
+            - name: efs-state-dir
+              mountPath: /var/run/efs
+            - name: efs-utils-config
+              mountPath: /var/amazon/efs
+            - name: efs-utils-config-legacy
+              mountPath: /etc/amazon/efs-legacy
```

Comment:
nit: should probably be `aws` instead, but that's not a big deal (and this is from upstream)

Reply:
This path is hardcoded somewhere in efs-utils, https://github.com/aws/efs-utils/blob/f3bb4a228a40bcf2e82fc28ab898d2b44ed8b05e/src/watchdog/__init__.py#L55
I'd prefer to follow upstream as close as possible.